### ISSUE TITLE
Multiple nodes and presence channels

### ISSUE THREAD

**Issue by billypap1** [soketi/soketi#608]
Hello, I have successfully deployed 2 instances with Soketi using the Cluster - Private network mode. 
I tested it with multiple users (2k concurrent) in public channel and 2k in presence channels. 
I noticed that when the users are on public channels, the load is shared between the 2 instances (~1000 on each one) but if the users are connected to presence channels only the master server receive the requests. 
So in other words, in public channels the 2 instances work like a load balancer but in presence channels only the master get the load and the other instance is standby. 
It should work like this or should load balance the load on presence channels also?
